projects/aca-content/assets/i18n/en.json

```diff
@@ -603,5 +603,30 @@
             "BOOKS-24PX": "file library",
             "BASELINE-LOCK-24PX": "locked file"
         }
+    },
+    "KNOWLEDGE_RETRIEVAL": {
+      "SEARCH": {
+        "RESULTS_PAGE": {
+          "QUERY_INPUT_PLACEHOLDER": "Would you like to ask anything else?",
+          "REFERENCED_DOCUMENTS_HEADER": "Referenced documents",
+          "LOADING_ERROR": "Hmm...something seems to have gone wrong.",
```

Comment:
Maybe API will return something more specific? With that users might be quite confused

Reply:
For now we have not integrated backend. Everything are mocks. If backend will return anything helpful we will change it during integration